src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -141,6 +146,11 @@ public TracerSettings(IConfigurationSource source)
         /// </summary>
         public IntegrationSettingsCollection Integrations { get; }
 
+        /// <summary>
+        /// Gets or sets the global tags, which are applied to all <see cref="Span"/>s.
+        /// </summary>
+        public ConcurrentDictionary<string, string> GlobalTags { get; set; }
```

Comment:
Might we expose this in the public API as an `IDictionary`? Or do we have specific reasons that we need to expose it as the `ConcurrentDictionary` type?

Reply:
I think you're right, I remember thinking the same thing but was getting it to work and didn't change it. I'll look at it.

Reply:
Yeah, let's expose as `IDictionary<string,string>`. Related: we make this change, we can also drop the cast to `ConcurrentDictionary<string, string>` in line 82 above.

Reply:
👍